profile/README.md

```diff
@@ -0,0 +1,22 @@
+## Hello from Norway's largest sports association 👋
+.. and welcome to our GitHub page! 🥳
+We are proud to be our country's largest sports association with more than 60 sports groups and 14 000 members.
+Being such a big organization sure requires some technical stuff, so that's why we're here :raising_hand_man: :raising_hand_woman:
+Our mission is not only to create solutions that we find useful ourselves, but also solutions that can make sports better in general.
+
+## Who are we?
+Our GitHub organization consists mostly of developers at NTNUI Sprint, the association's IT committee.
+We're also lucky to have some IT geniouses from the different sports groups as well 🤓
+
+## 💚 Contributing
+We're always open for contributions, whether you are affiliated with NTNUI or not.
+Have a look at the issue boards in each of the repositories, or contact us at sprint@ntnui.no if you're not sure how to get started.
+
+
+## 💎 Our gems at the moment
+### membership-system (currently private)
+### opptak
+### ntnui-tools
```

Comment:
Maybe link to the live version, especially when the repo is private? 😇

```suggestion
## 💎 Our gems at the moment
### membership-system (currently private), at [medlem.ntnui.no](https://medlem.ntnui.no)
### opptak, at [opptak.ntnui.no](https://opptak.ntnui.no)
### ntnui-tools, in an npm registry near you
```

Or something